projects/sandbox/train/train/train.py

```diff
@@ -47,6 +47,7 @@ def main(
     kernel_length: float,
     sample_rate: float,
     batch_size: int,
+    train_stop: float,
```

Comment:
Oh wow I didn't realize that we stopped passing the train start time to `train.py`, interesting... In either case, would it maybe make more sense to use that, then infer this like `train_stop = train_start + (1 - valid_frac) * background.shape[-1] / sample_rate`?

Reply:
Since we started querying glitches before the training background start time to increase numbers, I figured just splitting from the end of the training time made the most sense.

Reply:
I'm now realizing `train_stop` includes validation. Will have to update that logic

Reply:
Renaming variables for clarity and calculating this as below:
```
valid_start = (1 - valid_frac) * (train_val_stop - train_val_start) + train_val_start
train_gitches = glitches[times <= valid_start]
valid_glitches = glitches[times > valid_start]
```